MAUI/ListView/swiping.md

```diff
@@ -0,0 +1,628 @@
+{
+   if (e.Index == 1)
+      e.Cancel = true;
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Swiping Event
+ 
+The `SfListView.Swiping` event is raised while swiping an item is in progress. This event is triggered with `SwipingEventArgs`.
+
+The `Swiping` event provides the following properties in their arguments:
+
+ * `Index`: Defines the swiping item index.
+ * `DataItem`: Defines the underlying data associated with the swiped item as its arguments.
+ * `Direction`: Defines the swipe direction of the swiped item.
+ * `OffSet`: Defines the current swipe offset of the item being swiped.
+ * `Handled`: Defines that if it is `true`, current swipe offset value remains same for the swiped item until the `SwipeEnded` event is raised.
+
+The `Swiping` event used for the following use cases:
+
+* To maintain the current offset value for the swiped item till the `SwipeEnded` event gets called.
+* To hold the swipe view being swiping by setting the `Handled` property to `true` if swipe offset meet certain position.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml.cs" %}
+<syncfusion:SfListView x:Name="listView" ItemsSource="{Binding InboxInfo}" 
+                       Swiping="ListView_Swiping" />
+{% endhighlight %}
+{% highlight c# %}
+listView.Swiping += ListView_Swiping;
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight c# %}
+private void ListView_Swiping(object sender, SwipingEventArgs e)
+{
+   if (e.Index == 1 && e.OffSet > 70)
+       e.Handled = true;
+}
+{% endhighlight %}
+{% endtabs %}
+
+### SwipeEnded Event
+
+The `SfListView.SwipeEnded` event is fired when completing the swipe action. This event is triggered with `SwipeEndedEventArgs`.
+
+The `SwipeEnded` event provides the following properties in their arguments: 
+
+ * `Index`: Defines the swiping item index.
+ * `DataItem`: Defines the underlying data associated with the swiped item as its arguments. 
+ * `Direction`: Defines the swipe direction of the swiped item.
+ * `OffSet`: Defines the current swipe offset of the item being swiped.
+ 
+The `SwipeEnded` event is used for the following use cases:
+ 
+ * To insert the data or edit the data after swiped.
+ * To delete the item from view after swiping it to certain extent.
+ * To reset the swipe view automatically for the swiped item.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<syncfusion:SfListView x:Name="listView" ItemsSource="{Binding InboxInfo}" 
+                       SwipeEnded="ListView_SwipeEnded" />
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+listView.SwipeEnded += ListView_SwipeEnded;
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void ListView_SwipeEnded(object sender, SwipeEndedEventArgs e)
+{
+  if (e.Offset > 70)
+      listView.ResetSwipeItem();
+}
+{% endhighlight %}
+{% endtabs %}
+
+### SwipeReset Event
+
+
+`SwipeReset` event provides the following properties in their arguments:
```

Comment:
The `SwipeReset` event provides the following properties in their arguments:

Reply:
resolved